encode_utils/connection.py

```diff
@@ -171,10 +174,12 @@ def _set_dcc_mode(self,dcc_mode=False):
     dcc_mode = dcc_mode.lower()
     if dcc_mode not in eu.DCC_MODES:
       #: assume dcc_mode is a valid demo host
-      self.dcc_host = dcc_mode
-      self.dcc_url = 'https://' + dcc_mode + '/'
+      eu.DCC_MODES[dcc_mode] = {
```

Comment:
Is demo.encodedcc.org the only other possible host, or can there be several demo hosts? If several, I like your way of handling this. If just this extra one host, better we add it directly in __init__.DCC_MODES hash.

Reply:
All of our machines are either *.demo.encodedcc.org or *.production.encodedcc.org.  Test and production actually get mapped by our nginix proxy (e.g., we have a machine called v67x0-test-master.demo.encodedcc.org that is recieves the https: forwards from test.encodedcc.org.

Production only has the current and most recent production machines, and the 2nd one only for a few days after release.